src/Entity/AddressValidation/AVAddress.php

```diff
@@ -59,14 +67,24 @@ public function __construct(\SimpleXMLElement $xmlDoc)
         $this->addressClassification = isset($xmlDoc->AddressClassification) ? new AddressClassification($xmlDoc->AddressClassification) : null;
         $this->consigneeName = isset($xmlDoc->ConsigneeName) ? (string)$xmlDoc->ConsigneeName : '';
         $this->buildingName = isset($xmlDoc->BuildingName) ? (string)$xmlDoc->BuildingName : '';
-        $this->addressLine = isset($xmlDoc->AddressLine) ? (string)$xmlDoc->AddressLine : '';
-        $this->region = isset($xmlDoc->Region) ? (string)$xmlDoc->Regions : '';
+        if (isset($xmlDoc->AddressLine)) {
```

Comment:
Could we refactor this a bit? It feels a bit double for me.

I would say something like this will also do:

````
$var = 'addressLine' . ($i > 0 ? $i + 1 : '');
$this->{$var} = isset($xmlDoc->AddressLine[$i]) ? (string) $xmlDoc->AddressLine[$i] : '';
````